app/controllers/my_facilities/drug_stocks_controller.rb

```diff
@@ -14,11 +14,19 @@ def index
     @facilities = filter_facilities
       .includes(facility_group: :protocol_drugs)
       .where(protocol_drugs: {stock_tracked: true})
+
+    render && return if @facilities.empty?
+    # handle no facilities
```

Comment:
I think the code's fairly clear. We can remove this comment.

Reply:
Ha no, this was a TODO for us 😁 . Removed after handling in f9e2efdcbde68ce341d1b767c98bf1c3feb5d701.